python/cudf/cudf/core/dataframe.py

```diff
@@ -422,6 +423,7 @@ def __getattr__(self, key):
 
         raise AttributeError("'DataFrame' object has no attribute %r" % key)
 
+    @annotate("CUDF_DATAFRAME_GETITEM", color="blue", domain="cudf_python")
```

Comment:
Like I said on the other PR, the `CUDF` prefix is really unnecessary when using the `cudf_python` domain.

Reply:
Renamed them all